src/features/bumpkins/components/Trade.tsx

```diff
@@ -286,15 +306,10 @@ export const Trade: React.FC = () => {
     );
   }
 
-  if (gameState.matches("autosaving")) {
```

Comment:
I have removed this because it causes the listing form to close whenever an autosave fired. I can't think of a good reason for leaving this in. If you can think of something let me know.